reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py

```diff
@@ -19,6 +19,17 @@ def del_if_exists(d, field):
         del(d[field])
 
 
+def ee_get_json(mclient, ee_name):
+    all_ees = mclient.ee_list()
+    ee = list(filter(lambda x: x["name"] == ee_name, all_ees))
```

Comment:
Two lightning comment :) 
1. As a pythonist, I'd always encourage to use list comprehensions. IMHO, it is more clear.
2. In terms of performance, in cases where you provide a lambda, it is usually better to use list comprehension. The 'map', 'filter', etc, are better when you already have a defined function.

Reply:
Fixed